blocks/library/heading/test/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { query, getBlocks, unregisterBlock, setUnknownTypeHandler } from 'api';
+import { blockSettings } from '../';
+
+describe( 'heading block', () => {
+	afterEach( () => {
```

Comment:
Guessing `after` is probably more appropriate. This is needed since the block registers itself merely by being imported? Which is an unfortunate side effect. Seems like managing block registration lifecycle in tests will be a huge headache with the current setup. Simply forgetting one of these lifecycle handlers could have rippling effects on future tests, which is a maintainability nightmare.

I was wondering if, not unlike ideas proposed in #336, we could create the block registry as simply an instance of a Redux store, similar to global state. This way tests could be guaranteed to be isolated. Unsure yet where this initialization would occur in the app itself.

Alternatively, a more immediate solution might be to move the `registerBlock` call out of `library` and simply have the block export its settings. This doesn't set as good an example for third-party block implementers though, for which this pattern may or may not be the best.